Makefile

```diff
@@ -339,6 +360,40 @@ ifeq ($(CHECK_IF_PREINSTALLED), true)
 	fi
 endif
 
+check-release:
+	@echo "This Makefile rule checks that:"
+	@echo "- the VERSION and 'git describe' values are consistent"
+	@echo "- NEXT_RELEASE does not appear in the sources anymore"
+	@echo "For any more serious release checking, see howto/release.adoc."
+	@echo "(We only add output below when a check fails.)"
+	@echo
+	@$(MAKE) --silent check-release-VERSION-git-describe
+	@$(MAKE) --silent check-release-NEXT_RELEASE
```

Comment:
I use recursive make when I want to force ordering. My understanding of make semantics is that targets given in a single call may be executed in any order in presence of `-jN`. Is this wrong?